mackerel-plugin-mysql/lib/mysql.go

```diff
@@ -754,33 +754,33 @@ func parseInnodbStatus(str string, trxIDHexFormat bool, p *map[string]float64) {
 		record := strings.Fields(line)
 
 		// Innodb Semaphores
-		if strings.Index(line, "Mutex spin waits") == 0 {
+		if strings.HasPrefix(line, "Mutex spin waits") {
 			increaseMap(p, "spin_waits", record[3])
 			increaseMap(p, "spin_rounds", record[5])
 			increaseMap(p, "os_waits", record[8])
 			continue
 		}
-		if strings.Index(line, "RW-shared spins") == 0 && strings.Index(line, ";") > 0 {
+		if strings.HasPrefix(line, "RW-shared spins") && strings.Contains(line, ";") {
```

Comment:
Precisely speaking `strings.Contains` is exactly equal to `strings.Index(...) >= 0`, not `> 0` but `strings.Contains` is more readable and does not change the behavior of parsing mysql innodb status output.